src/oc/web/ws/notify_client.cljs

```diff
@@ -59,6 +60,17 @@
                      (not (:udt-next-reconnect @@ch-state)))
             (utils/after (* 10 1000)
              (reconnect @last-ws-link (j/user-id)))))
+        (or (= rep :chsk/closed)
```

Comment:
Could we clean up some of the code repeat here. Maybe a shared disconnect handler between the web socket clients?